### PR TITLE
Extend tests_to_layer map with all test cases from a community

### PR DESCRIPTION
**Pull request by bratashX**
Extend tests_to_layer map with all test cases from a community. 
All TC  on `a1849ff51513bbe5a111754efe792f34033fef18` _(202205_hedgehog_dev)_  .

### CODE REVIEW

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 1]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+platform_tests.test_reboot#test_warm_reboot: MGMT
+platform_tests.test_reboot#test_watchdog_reboot: MGMT
+platform_tests.test_reload_config#test_reload_configuration: MGMT
+platform_tests.test_reload_config#test_reload_configuration_checks: MGMT
+platform_tests.test_sensors#test_sensors: MGMT
+platform_tests.test_sequential_restart#test_restart_swss: MGMT
+platform_tests.test_sequential_restart#test_restart_syncd: MGMT
+platform_tests.test_service_warm_restart#test_service_warm_restart: MGMT
+platform_tests.test_xcvr_info_in_db#test_xcvr_info_in_db: MGMT
+portstat.test_portstat#test_portstat_clear: MGMT
```
Comment (nazar-garmadiy) [inline tests/hedgehog/mapping_test_to_group.yaml]: The portstat is related to port state and counters. Might be L2 and MGMT

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 2]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+portstat.test_portstat#test_portstat_delete_all: MGMT
+portstat.test_portstat#test_portstat_delete_tag: MGMT
+portstat.test_portstat#test_portstat_display_all: MGMT
+portstat.test_portstat#test_portstat_no_exceptions: MGMT
+portstat.test_portstat#test_portstat_period: MGMT
+process_monitoring.test_critical_process_monitoring#test_monitoring_critical_processes: MGMT
+qos.test_buffer#test_buffer_deployment: MGMT
```
Comment (nazar-garmadiy) [inline tests/hedgehog/mapping_test_to_group.yaml]: I would say that QoS is related to L2 (or higher).

**Review by nazar-garmadiy — COMMENTED** [review]
Reviewed lines from 500 - 1002. Overall looks good.

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 3]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
 dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_start_with_uplinks_down: MGMT
 dhcp_relay.test_dhcpv6_relay#test_dhcpv6_relay_counter: MGMT
 dhcp_relay.test_dhcpv6_relay#test_interface_binding: MGMT
+drop_packets.test_configurable_drop_counters#test_dip_link_local: L3
```
Comment (maksymhedeon) [inline tests/hedgehog/mapping_test_to_group.yaml:175]: Not all "drop counters" tests are MGMT?

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 4]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+everflow.test_everflow_testbed.TestEverflowV4IngressAclEgressMirror#test_everflow_dscp_with_policer: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclEgressMirror#test_everflow_neighbor_mac_change: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclEgressMirror#test_everflow_remove_unused_ecmp_next_hop: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclEgressMirror#test_everflow_remove_used_ecmp_next_hop: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclIngressMirror#test_everflow_basic_forwarding: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclIngressMirror#test_everflow_dscp_with_policer: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclIngressMirror#test_everflow_neighbor_mac_change: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclIngressMirror#test_everflow_remove_unused_ecmp_next_hop: MGMT
+everflow.test_everflow_testbed.TestEverflowV4IngressAclIngressMirror#test_everflow_remove_used_ecmp_next_hop: MGMT
+fdb.test_fdb#test_fdb: L3
```
Comment (maksymhedeon) [inline tests/hedgehog/mapping_test_to_group.yaml]: FDB is L2

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 5]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+fdb.test_fdb#test_self_mac_not_learnt: L3
+fdb.test_fdb_mac_expire.TestFdbMacExpire#testFdbMacExpire: L3
+fib.test_fib#test_basic_fib: L3
+fib.test_fib#test_hash: L3
+generic_config_updater.test_aaa#test_tacacs_server_tc3_suite: MGMT
```
Comment (bratashX) [inline tests/hedgehog/mapping_test_to_group.yaml:272]: generic_config_updater is MGMT

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 6]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+generic_config_updater.test_vlan_interface#test_vlan_interface_tc1_suite: L2
+generic_config_updater.test_vlan_interface#test_vlan_interface_tc2_incremental_change: L2
 http.test_http_copy#test_http_copy: MGMT
+iface_namingmode.test_iface_namingmode.TestConfigInterface#test_config_interface_ip: L3
+iface_namingmode.test_iface_namingmode.TestConfigInterface#test_config_interface_speed: L3
+iface_namingmode.test_iface_namingmode.TestConfigInterface#test_config_interface_state: L3
+iface_namingmode.test_iface_namingmode.TestNeighbors#test_show_arp: L3
+iface_namingmode.test_iface_namingmode.TestNeighbors#test_show_ndp: MGMT
+iface_namingmode.test_iface_namingmode.TestShowIP#test_show_ip_interface: L3
+iface_namingmode.test_iface_namingmode.TestShowIP#test_show_ip_route_v4: L3
```
Comment (bratashX) [inline tests/hedgehog/mapping_test_to_group.yaml]: iface_namingmode is  MGMT

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 7]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+pc.test_lag_2#test_lag: L2
+pc.test_lag_2#test_lag_db_status: L2
+pc.test_lag_2#test_lag_db_status_with_po_update: L2
+pc.test_lag_member#test_lag_member_status: L2
+pc.test_lag_member#test_lag_member_traffic: L2
+pc.test_po_cleanup#test_po_cleanup: L2
+pc.test_po_cleanup#test_po_cleanup_after_reload: L2
 pc.test_po_update#test_po_update: MGMT
 pc.test_po_update#test_po_update_io_no_loss: MGMT
+pfc_asym.test_pfc_asym#test_pfc_asym_off_rx_pause_frames: MGMT
```
Comment (bratashX) [inline tests/hedgehog/mapping_test_to_group.yaml]: pfc_asym is L2

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 8]
```diff
@@ -19,26 +172,625 @@ dhcp_relay.test_dhcpv6_relay#test_dhcp_relay_default: MGMT
...
+qos.test_tunnel_qos_remap#test_tunnel_decap_dscp_to_queue_mapping: MGMT
+qos.test_tunnel_qos_remap#test_xoff_for_pcbb: MGMT
 radv.test_radv_ipv6_ra#test_radv_router_advertisement: L3
 radv.test_radv_ipv6_ra#test_solicited_router_advertisement: L3
 radv.test_radv_ipv6_ra#test_solicited_router_advertisement_with_m_flag: L3
 radv.test_radv_ipv6_ra#test_unsolicited_router_advertisement_with_m_flag: L3
+radv.test_radv_restart#test_radv_swss: L3
+read_mac.test_read_mac_metadata#test_read_mac_metadata: L2
+restapi.test_restapi#test_check_reset_status: L3
+restapi.test_restapi#test_create_interface: L3
```
Comment (bratashX) [inline tests/hedgehog/mapping_test_to_group.yaml]: MGMT?

File: tests/hedgehog/mapping_test_to_group.yaml [hunk 9]
```diff
@@ -102,5 +924,79 @@ test_hedgehog_smoke#test_syncd_smoke: MGMT
 test_hedgehog_smoke#test_syslog_smoke: MGMT
 test_hedgehog_smoke#test_vrrp_smoke: L2
 test_interfaces#test_interfaces: L3
+test_procdockerstatsd.py#test_verify_redisexport: MGMT
+test_procdockerstatsd.py#test_verify_status: MGMT
+testbed_setup.test_populate_fdb#test_populate_fdb: L3
```
Comment (bratashX) [inline tests/hedgehog/mapping_test_to_group.yaml]: ?

File: tests/hedgehog/mapping_test_to_group.yaml (hunk 3 above)
Comment (bratashX) [reply tests/hedgehog/mapping_test_to_group.yaml:175]: I think better categorizes drop counters" tests due to drop reason L2 or L3
It discased with @nazar-garmadiy

File: tests/hedgehog/mapping_test_to_group.yaml (hunk 1 above)
Comment (bratashX) [reply tests/hedgehog/mapping_test_to_group.yaml]: Changed to L2

**Review by nazar-garmadiy — APPROVED** [review]

**Pull request merged**